multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.119.0-quartz.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        logicalFilePath="org/cloudfoundry/multiapps/controller/core/db/changelog/db-changelog-1.119.0-quartz.xml"
```

Comment:
This line is not necessary (see https://docs.liquibase.com/concepts/advanced/logicalfilepath.html).